commands/auto/autonomousaction.py

```diff
@@ -0,0 +1,117 @@
+from typing import Dict, List
+from commands2 import (
+    Command,
+    CommandBase,
+    FunctionalCommand,
+    ParallelCommandGroup,
+    ParallelDeadlineGroup,
+    SequentialCommandGroup,
+    WaitCommand,
+)
+from pathplannerlib import PathPlannerTrajectory
+from wpilib import DataLogManager
+
+from commands.auto.autohelper import trajectoryFromFile
+from commands.auto.followtrajectory import FollowTrajectory
+from commands.resetdrive import ResetDrive
+from subsystems.drivesubsystem import DriveSubsystem
+
+
+class AutonomousRoutine(SequentialCommandGroup):
+    markerMap: Dict[str, Command]
+
+    def __init__(
+        self,
+        drive: DriveSubsystem,
+        name: str,
+        simultaneousCommands: List[Command],
+    ):
+        self.name = name
+        self.markerMap = {  # later todo: actual implementation
+            "store": WaitCommand(2),
+            "top": WaitCommand(2),
+            "mid": WaitCommand(2),
+            "hybrid": WaitCommand(2),
+            "engage": WaitCommand(5),
+            "intake": WaitCommand(0.25),
+            "outtake": WaitCommand(0.25),
+        }
+        self.paths = trajectoryFromFile(name)
```

Comment:
Do the files get manually updated on the Rio?  Or does the robotpy installer take care of that?

Reply:
robotpy-installer is very dumb -- all files are dumped and therefore the trajectory files are dumped

Reply:
hey whatever works